docs/network.md

```diff
@@ -57,7 +57,7 @@ autodetect the native token symbol and decimal places, the chain domain
 separation context and registered ParaTimes.
 
 ```
-oasis network add-local testnet_local unix:/serverdir_testnet/internal.sock
+oasis network add-local testnet_local unix:/node/data_testnet/internal.sock
```

Comment:
I think it would make more sense to be `/node_testnet/data` instead?

Reply:
```suggestion
oasis network add-local testnet_local unix:/node_testnet/data/internal.sock
```
I also believe `/node_testnet/data` would be better. Because Testnet and Mainnet nodes use a different configuration (`etc/`), different runtimes (`runtimes/`), and different binary versions (`bin/`)...